internal/services/storage/storage_account_resource.go

```diff
@@ -214,7 +202,11 @@ func resourceStorageAccount() *pluginsdk.Resource {
 					},
 				},
 			},
-
+			"cross_tenant_replication": {
```

Comment:
should this be
```suggestion
			"cross_tenant_replication_enabled": {
```

Reply:
I had this feeling I forgot something ... :)